dev-requirements.txt

```diff
@@ -3,9 +3,10 @@ pytest
 pytest-cov
 coverage
 mock
-sphinx<1.7
+sphinx>=2.0.0
 sphinx_rtd_theme
-jinja2
+jinja2<3
+markupsafe==2.0.1
```

Comment:
CI is not happy with these changes (test-2.7 and test-3.5) and according to the PR title and commits message it shouldn't be here.